pkg/apis/performanceprofile/v2/performanceprofile_validation.go

```diff
@@ -216,11 +216,11 @@ func validateNoIntersectionExists(lists *components.CPULists, allErrs field.Erro
 func (r *PerformanceProfile) validateSelectors() field.ErrorList {
 	var allErrs field.ErrorList
 
-	if r.Spec.MachineConfigLabel != nil && len(r.Spec.MachineConfigLabel) > 1 {
+	if len(r.Spec.MachineConfigLabel) > 1 {
```

Comment:
is this part of the intended change ?

Reply:
Yes, it is. There was a linter warning to simplify the code.